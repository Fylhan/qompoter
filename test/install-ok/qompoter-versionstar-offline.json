{
	"name": "test/install-ok",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/fylhan": "v1.*"
	}
}
