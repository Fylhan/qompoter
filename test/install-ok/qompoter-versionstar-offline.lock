{
  "name": "install-ok/qompoter-versionstar-offline",
  "date": "2016-12-02T18:39:33+01:00",
  "require": {
    "cp/acme-cp": {"version": "v1.0-lib", "type": "qompoter-fs", "url": "", "md5sum": "115f3cd6f4a5dec31fb31b9cf39fa855"}
    "cp/fylhan": {"version": "v1.1.3-beta", "type": "qompoter-fs", "url": "", "md5sum": "9d57702586a06dcd5de4d717d936c9d6"}
  }
}
