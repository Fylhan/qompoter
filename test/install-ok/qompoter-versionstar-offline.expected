Qompoter
======== install

* cp/fylhan v1.*
  Downloading sources...
  Selected version: v1.1.3-beta
  done

* cp/acme-cp v1.*-lib
  Downloading lib...
  Selected version: v1.0-lib
  done

done
